v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -221,7 +221,7 @@ public interface ElasticsearchWriteOptions extends PipelineOptions {
       optional = true,
       description = "Use partial updates",
       helpText =
-          "Whether to use partial updates (update rather than create or index, allowing partial docs) with Elasticsearch requests. Default: 'false'")
+          "Whether to use partial updates (update rather than create or index, allowing partial docs) with Elasticsearch requests. Default: `false`.")
```

Comment:
```suggestion
          "Whether to use partial updates (update rather than create or index, allowing partial documents) with Elasticsearch requests. Defaults to: false.")
```